plasma_framework/truffle-config.js

```diff
@@ -92,15 +92,14 @@ module.exports = {
     // Configure your compilers
     compilers: {
         solc: {
-            // version: "0.5.1",    // Fetch exact version from solc-bin (default: truffle's version)
-            // docker: true,        // Use "0.5.1" you've installed locally with docker (default: false)
-            // settings: {          // See the solidity docs for advice about optimization and evmVersion
-            //  optimizer: {
-            //    enabled: false,
-            //    runs: 200
-            //  },
-            //  evmVersion: "byzantium"
-            // }
+            version: '0.5.11',
+            settings: {
+                optimizer: {
+                    enabled: true,
+                    runs: 200,
```

Comment:
Should we set `runs: 1` to optimise for size?

Reply:
hmm, I don't know which is better. But if it is deploy-able, do we need to keep optimize for deploy cost and not optimize for transaction?

Reply:
I'm surprised it's deployable! :)
But yeah, let's leave it like this and change it later if necessary